src/plugins/blocksmith/components/imgur-post-block-preview/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Message from '@gdbots/pbj/Message';
+import ImgurPostBlockEmbed from './embed';
+
+class ImgurPostBlockPreview extends Component {
+  constructor(props) {
+    super(props);
+    this.createEmbed = this.createEmbed.bind(this);
+    this.loadScript = this.loadScript.bind(this);
+  }
+
+  loadScript() {
+    const embedScript = document.getElementById('globalImgurEmbedScriptTagId');
```

Comment:
@jalopez13 make this id a constant `const IMGUR_EMBED_SCRIPT_ID = 'imgur-embed-script'`

Reply:
@efjacobson 

Added const IMGUR_EMBED_SCRIPT_ID